### ISSUE TITLE
Converting a TensorFlow* Model

### ISSUE THREAD

**Issue by DucLong06** [opencv/dldt#409]
I want to convert my 1 tensorflow model to IR currently I am following the instructions here:
https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow.html

The model I use is meta graph and ubuntu 16.04

I ran the line deflected:
python3 mo_tf.py --input_meta_graph <INPUT_META_GRAPH> .meta

then it will get an error:
[ERROR] Exception occurred during running replacer "None" (<class 'extensions.front.tf.assign_elimination.AssignSubElimination'>): Data flow edge coming out of AssignSub node model_0 / resnet_v1_50 / block4 / unit_1 / bottleneck_v1 / shortcut / BatchNorm / AssignMovingAvg

Can you guys please help me? thanks everyone

**Comment by jgespino** [comment]
Hi @DucLong06 

Is your model a custom trained model? Did you freeze the model? What base model did you use?

You need to use a couple more options with the mo_tf.py, such as:
--tensorflow_custom_operations_config_update
--tensorflow_object_detection_api_pipeline_config
--input_shape
--reverse_input_channels

If possible, please share your model and necessary files for me to reproduce.

Regards,
Jesus

**Comment by DucLong06** [comment]
hi @jgespino 
Can you help me convert this model:
https://drive.google.com/drive/folders/136eex78WF5S4vKyos4y3oj9L2BD0BhAE?usp=sharing
If possible, you can help to list in detail the steps to convert it
Thanks very much

**Comment by jgespino** [comment]
Hi @DucLong06,

You will need to freeze the model to Pb format first using a freeze python script like [this one](https://blog.metaflow.fr/tensorflow-how-to-freeze-a-model-and-serve-it-with-a-python-api-d4f3596b3adc?gi=3f4ef5827be6). However, you will need to know the `output_node_name`. 

I tried to open your meta file but was not able to open it successfully using Netron. Could you please provide the name of the output node?

Regards,
Jesus

**Comment by DucLong06** [comment]
Hi @jgespino
Thanks you but I'm also having that problem I'm referencing here: https://stackoverflow.com/questions/45864363/tensorflow-how-to-convert-meta-data-and-index-model-files-into-one- graph-pb 
and Do you have a document about it ? I want to reference it .

**Comment by jgespino** [comment]
Hi @DucLong06 

You need to know the name of the output nodes in order to freeze the TensorFlow Model. Try using Netron or TensorFlow Summarize Graph utility to explore the model. 

I'd suggest posting your question on the TensorFlow Github as this appears to be a TensorFlow specific question and see if they have any other recommendations.

Regards,
Jesus

**Comment by DucLong06** [comment]
thank so much @jgespino
